src/main/java/org/jabref/logic/importer/fileformat/MsBibImporter.java

```diff
@@ -85,4 +92,29 @@ public String getDescription() {
         return "Importer for the MS Office 2007 XML bibliography format.";
     }
 
+    /**
+     * DocumentBuilderFactory makes a XXE safe Builder factory from dBuild. If not supported by current
+     * XML then returns original builder given and logs error.
+     * @param dBuild | DocumentBuilderFactory to be made XXE safe.
+     * @return If supported, XXE safe DocumentBuilderFactory. Else, returns original builder given
+     */
+    private DocumentBuilderFactory makeSafeDocBuilderFactory(DocumentBuilderFactory dBuild) {
+        String FEATURE = null;
+        try {
+            FEATURE = "http://apache.org/xml/features/disallow-doctype-decl";
```

Comment:
Just some minor improvemtns: I would suggest extracting both urls to two separate variables and make them private static final as constants.